lib/filepicker_rails/filepicker_image_url.rb

```diff
@@ -0,0 +1,141 @@
+# Creates the full path of the image to the specified `url` accepts optional
+# `options` hash for configuration.
+#
+# #### Constructor
+#
+# @param [String] url                       Base filepicker url
+# @param [Hash] options
```

Comment:
I think we can remove the options here and make a reference to the `filepicker_image_url` we already do this on the [`filepicker_image_tag`](https://github.com/SchoolKeep/filepicker-rails/blob/public-policy-image-url-generation/app/helpers/filepicker_rails/application_helper.rb#L99) this way we avoid duplicated docs.